lib/srv/db/postgres/users.go

```diff
@@ -163,6 +173,13 @@ const (
 	// teleportAutoUserRole is the name of a PostgreSQL role that all Teleport
 	// managed users will be a part of.
 	teleportAutoUserRole = "teleport-auto-user"
+
+	// sqlStateUserDropped is the state returned by the delete procedure
+	// indicating the user was dropped.
+	sqlStateUserDropped = "TP001"
+	// sqlStateUserDropped is the state returned by the delete procedure
+	// indicating was deactivated.
+	sqlStateUserDeactivated = "TP002"
```

Comment:
Could we use the same state sas MySQL? Probably can move these state constants to common too.

Reply:
Done on commit [4cf70ef0e8f422e59f3831816d6af88439f2c2c8](https://github.com/gravitational/teleport/pull/32792/commits/4cf70ef0e8f422e59f3831816d6af88439f2c2c8).

Reply:
Thanks!!